packages/shared/src/enterprise/licenseUtil.ts

```diff
@@ -147,6 +158,19 @@ export interface LicenseInterface {
     discountMessage?: string; // The message of the discount
     hasPaymentMethod?: boolean;
   };
+  orbSubscription?: {
+    id: string;
+    customerId: string;
+    qty: number;
+    trialEnd: Date | null;
+    status: Stripe.Subscription.Status;
+    current_period_end: number;
+    cancel_at: number | null;
+    canceled_at: number | null;
+    cancel_at_period_end: boolean;
+    planId: string;
+    hasPaymentMethod: boolean;
```

Comment:
We'll probably also want to add `portalUrl` here.

Reply:
This whole license object is visible to everyone in the org, even readonly users.  Is it safe to expose the portal URL like that?  Or should we use the existing `/subscriptions/manage` endpoint that fetches the URL from the license server after doing permission checks?

Reply:
Ah - I see. I forgot we pass the full `license` to the front-end. My initial thought was to only share the portal URL if the user had permission, but that doesn't fix us sharing the full license.

I'm good with updating `subscription/manage`, to handle both stripe and orb subscriptions.